docs/3.0.0-beta.x/guides/auth-request.md

```diff
@@ -0,0 +1,172 @@
+# Authenticated request
+
+In this guide we will see how you can request the API as an authenticated user.
```

Comment:
`In this guide you will see how you can request the API as an authenticated user.`